libs/csm/rest/csm_rest_query_deployment.py

```diff
@@ -19,7 +19,8 @@
 from http import HTTPStatus
 import yaml
 
-from config import PROV_TEST_CFG
+from config import CSM_REST_CFG, PROV_TEST_CFG
```

Comment:
CSM_REST_CFG is not required
It available via self.config

Reply:
modified.